src/components/ChartTodo/ChartTodo.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { Paper } from '@material-ui/core';
+import {BarChart,CartesianGrid ,XAxis,YAxis,Bar , ResponsiveContainer  , Tooltip, Text} from 'recharts'
+import classes from './ChartTodo.module.css';
+import axios from '../../axios-ListData';
+import withErrorHandler from '../../hoc/withErrorHandler/withErrorHandler';
+
+
+class ChartTodo extends Component {
+
+
+    state= {
+        todoList : [],
+
+    }
+
+    componentDidMount () {
+        axios.get('/ListTodo.json')
+             .then(response => {
+                let updateTodoData = [] ;
+                let todoData = Object.keys(response.data).map(item => {
+                    return {...response.data[item]  ,id : item}  })
+                    for( let key in todoData) {
+                        updateTodoData.push(todoData[key])
+                    }
+                    const res = Object.values(updateTodoData.reduce((item, {Date}) => {
+                        item[Date] = item[Date] || {Date,count: 0};
+                        item[Date].count++;
+                        return item;
+                      }, Object.create(null)));
+                      
+                      console.log(res)
+                this.setState({
+                    todoList : res ,
+                })
+             }).catch(err=>{
+                 console.log(err)
+             })
+    }
```

Comment:
Please Fix the code indentation.
Can we map the object directly in one step?

Reply:
I'm Fix the code, updateTodoData and todo data it's the same but I forgot to remove it, I will change this code, but the idea I need to change object{key{object}} to a new object with different behavior like this  object{Date, count}.